.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Run tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Use Docker in rootless mode (To solve "Permission denied" error when removing test_app directory)
+        uses: ScribeMD/rootless-docker@0.2.2
```

Comment:
More explanation can be found here: https://github.com/marketplace/actions/rootless-docker
In short: all docker commands (i.e. creating contracts using DUNE) are executed as root, which causes access issues. Particulary `test_app` directory is owned by root, not by user `runner` which executes the tests.